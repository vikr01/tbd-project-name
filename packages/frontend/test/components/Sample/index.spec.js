import React from 'react';
import renderer from 'react-test-renderer';

describe('Sample component', () => {
  test('should match exact Sample snapshot', async () => {
    const {
      default: Sample,
    } = await import('toms-shuttles-frontend/src/components/Sample');
    const sample = <Sample />;

    const tree = renderer.create(sample);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
