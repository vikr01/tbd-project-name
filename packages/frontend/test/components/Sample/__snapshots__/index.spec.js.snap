// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample component should match exact Sample snapshot 1`] = `
<div>
  Hello!
</div>
`;
